new-lamassu-admin/src/pages/Notifications/Notifications.js

```diff
@@ -83,6 +108,22 @@ const Notifications = ({
     setEditingKey(state ? key : null)
   }
 
+  const twilioSave = it => {
+    setError(null)
+    R.compose(save(null), toNamespace('sms'))({ active: true })
+    return saveAccount({
+      variables: { accounts: { twilio: it } }
+    })
```

Comment:
Doesn't this leave us open to a success in the first save, and a failure in the second one? Why not create a mutation that does both at the same time?